compaction/compaction.hh

```diff
@@ -43,6 +43,7 @@ public:
 
 sstring compaction_name(compaction_type type);
 compaction_type to_compaction_type(sstring type_name);
+std::string_view to_string(compaction_type type);
```

Comment:
This needs a comment to explain the difference v.s. `compaction_name()`.

Reply:
np

Reply:
added documentation in next version.